### ISSUE TITLE
[Bug]: mo panic with index out of range [1] with length 1 when run tpcc benchmark test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6190]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):9130c990223a1d709051cff79b27ce3de65aad67
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

This is an occasional problem, when run tpcc benchmark test with 1-warehouse-10-termimals,  sometimes, mo will panic,

the error logs are as followings:

2022/11/02 14:33:46.523653 +0800 DEBUG moengine/helper.go:39 try set 85C77F6CFC6C407BAB942FD2A4DA6D19 txn access info to t0(true) u1(true) r0(true),
2022/11/02 14:33:46.523803 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.525687 +0800 DEBUG containers/utils.go:229 sie : 8997
2022/11/02 14:33:46.528545 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.528592 +0800 DEBUG containers/utils.go:229 sie : 8720
2022/11/02 14:33:46.529023 +0800 DEBUG containers/utils.go:229 sie : 399
2022/11/02 14:33:46.529264 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.531004 +0800 DEBUG containers/utils.go:229 sie : 8931
2022/11/02 14:33:46.539657 +0800 DEBUG containers/utils.go:229 sie : 17466
2022/11/02 14:33:46.539876 +0800 DEBUG containers/utils.go:229 sie : 8906
2022/11/02 14:33:46.541798 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.541883 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.549631 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.551190 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.553218 +0800 DEBUG containers/utils.go:229 sie : 9260
2022/11/02 14:33:46.554288 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.556838 +0800 DEBUG txnimpl/localseg.go:215 Offset=0, ToAppend=13, EstimateSize=1092
2022/11/02 14:33:46.558561 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.558596 +0800 DEBUG txnimpl/localseg.go:224 Appended: 13, Space:9931
2022/11/02 14:33:46.558756 +0800 INFO frontend/mysql_cmd_executor.go:2841 time of Exec.Run : 318.0835ms
2022/11/02 14:33:46.558786 +0800 INFO frontend/mysql_cmd_executor.go:2847 time of SendResponse 42ns
2022/11/02 14:33:46.558842 +0800 INFO frontend/util.go:474 query trace status {"connection_id": 26353, "statement": "execute __mo_stmt_id_30 using @__mo_stmt_var_0,@__mo_stmt_var_1,@__mo_stmt_var_2,@__mo_stmt_var_3", "status": "success"}
2022/11/02 14:33:46.558856 +0800 DEBUG frontend/routine.go:170 connection id 26353 , the time of handling the request 318.566166ms
2022/11/02 14:33:46.560403 +0800 WARN containers/utils.go:164 [Moengine] {"operation": "movecToBytes", "operand": "Col length is 0"}
2022/11/02 14:33:46.563614 +0800 DEBUG containers/utils.go:229 sie : 272
2022/11/02 14:33:46.564218 +0800 INFO frontend/mysql_cmd_executor.go:2841 time of Exec.Run : 481.683ms
2022/11/02 14:33:46.564257 +0800 INFO frontend/mysql_cmd_executor.go:2847 time of SendResponse 42ns
2022/11/02 14:33:46.564324 +0800 INFO frontend/util.go:474 query trace status {"connection_id": 26354, "statement": "execute __mo_stmt_id_8 using @__mo_stmt_var_0,@__mo_stmt_var_1,@__mo_stmt_var_2,@__mo_stmt_var_3,@__mo_stmt_var_4", "status": "success"}
2022/11/02 14:33:46.564345 +0800 DEBUG frontend/routine.go:170 connection id 26354 , the time of handling the request 482.478333ms
panic: runtime error: index out of range [1] with length 1

goroutine 667639 [running]:
github.com/matrixorigin/matrixone/pkg/common/bitmap.(*Bitmap).AddMany(...)
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/common/bitmap/bitmap.go:187
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers.UnmarshalToMoVec({0x1072c69a8, 0x14069cbb440})
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/vm/engine/tae/containers/utils.go:90 +0x34c
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/moengine.(*txnBlock).Read(0x1401f007e28, {0x140455f5830, 0x9, 0x9}, {0x1407ddace38?, 0x10664f740?, 0x2?}, {0x140a2896460, 0x9, 0x1403056bbf0?})
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/vm/engine/tae/moengine/block.go:50 +0x2a4
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/moengine.(*txnReader).Read(0x140561c6240, {0x140455f5830, 0x9, 0x9}, 0x104fefe00?, 0x1404be20a80)
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/vm/engine/tae/moengine/reader.go:55 +0x178
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).Run(0x1401f007f10, {0x10722fa48, 0x140561c6240}, 0x14025f44000)
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/vm/pipeline/pipeline.go:68 +0x150
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).Run(0x14023cf8780?, 0x1403f8209c0?)
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/sql/compile/scope.go:55 +0xa8
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func1(0x14035640708?)
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/sql/compile/scope.go:73 +0x60
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
        /Users/sudong/Program/MatrixOrigin/matrixone/pkg/sql/compile/scope.go:68 +0x174

**Comment by aptend** [comment]
@aressu1985 how to `run tpcc benchmark test with 1-warehouse-10-termimals`?  i add some logs and try to repro

**Comment by aptend** [comment]
provide aressu1985 with a logged branch to repro this issue tonight, it is expected that we can know which database and table cause this problem

**Comment by XuPeng-SH** [comment]
It's hard to reproduce. Change it to s0

**Comment by florashi181** [comment]
has it been reproduced now？

**Comment by aptend** [comment]
> has it been reproduced now？

Not yet

**Comment by aptend** [comment]
have no time on this today

**Comment by aptend** [comment]
no repro. I ran three times warehouse=1 terminals=10 runMins=10

**Comment by aptend** [comment]
no repro. I ran three times warehouse=5 terminals=10 runMins=5

**Comment by aptend** [comment]
@aressu1985 no repro again, please consider downgrading or closing

**Comment by aressu1985** [comment]
fixed:9cb1d8b58efea619a19f41a0cdaf0743b78d195f
